valkey.conf

```diff
@@ -386,6 +386,11 @@ databases 16
 # ASCII art logo in startup logs by setting the following option to yes.
 always-show-logo no
 
+# To avoid logging personal identifiable information (PII) into server log file,
+# uncomment the following:
+#
+# hide-client-log yes
```

Comment:
Should we enable this by default? Otherwise end users will maybe use their user data in stack traces?